moveos/moveos-stdlib/moveos-stdlib/sources/storage_context.move

```diff
@@ -33,4 +40,20 @@ module moveos_std::storage_context {
         &mut this.object_storage
     }
 
+    #[test_only]
+    /// Create a StorageContext for unit test
+    public fun test_context(sender: address): StorageContext {
+        let tx_context = tx_context::test_context(sender);
+        let object_storage = object_storage::new(&mut tx_context);
```

Comment:
The global object storage's table handle should be 0x0. We can export a friend function in object_storage, like `new_with_id` in TypeTable.